packages/storybook/src/nimble/list-option/list-option.stories.ts

```diff
@@ -0,0 +1,60 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import {
+    apiCategory,
+    createUserSelectedThemeStory
+} from '../../utilities/storybook';
+import { hiddenWrapper } from '../../utilities/hidden';
+
+const metadata: Meta = {
+    title: 'Internal/ListOption',
+    parameters: {
+        docs: {
+            description: {
+                component:
+                    'The `nimble-list-option` component is used to define the options that are available to the `nimble-select` component.'
```

Comment:
At some point we should embed this story in the combobox mdx too. Your call between these options:
1. do this later and leave this as-is, hope we remember later to update its contents to be agnostic to what parent it's in
2. do this later but update it now to be agnostic about what parent it's hosted in
3. do this now

Regardless of which option you pick, we should use constants for the tag names.

Reply:
Made an attempt at making it agnostic.